pyproject.toml

```diff
@@ -81,9 +81,16 @@ dev-pytest = [
   "pytest-asyncio == 0.24.0",
   "async-solipsism == 0.7",
   "hypothesis == 6.112.2",
+  "frequenz-quantities[marshmallow]",
 ]
+
+marshmallow = [
+  "marshmallow == 3.22.0",
+  "marshmallow-dataclass == 8.7.1",
+]
```

Comment:
These dependencies need to be wide, as this is a library, otherwise it forces to pin the dependencies to all project using this library.

Reply:
Updated